### PR TITLE
fix: syntax errors

### PR DESCRIPTION
**Pull request by OnkarRuikar**
The PR:
- fixes syntax errors
- trailing spaces were removed
- adds `.prettierignore` files so Prettier can be run
-

### CODE REVIEW

File: css/styling-text/typesetting-a-homepage-finished/original-font-files/Lovato Light-Webfont/Reference/How_to_use_webfonts.html [hunk 1]
```diff
@@ -220,11 +220,10 @@ <h3>Fonts not loading in IE9</h3>
...
 			</div>
-			
+
 		</div>
-	
+
 	</div>
-	
-</div>
+
```
Comment (OnkarRuikar) [inline css/styling-text/typesetting-a-homepage-finished/original-font-files/Lovato Light-Webfont/Reference/How_to_use_webfonts.html:227]: Removed extra closing `</div>`.

File: html/forms/form-validation/full-example.html [hunk 2]
```diff
@@ -36,16 +36,14 @@
...
-      <fieldset>
-        <legend>Do you have a driver's license?<span aria-label="required">*</span></legend>
-        <!-- While only one radio button in a same-named group can be selected at a time,
-             and therefore only one radio button in a same-named group having the "required"
-             attribute suffices in making a selection a requirement -->
-        <input type="radio" required name="driver" id="r1" value="yes"><label for="r1">Yes</label>
-        <input type="radio" required name="driver" id="r2" value="no"><label for="r2">No</label>
-      </fieldset>
-    </p>
+    <fieldset>
```
Comment (OnkarRuikar) [inline html/forms/form-validation/full-example.html:39]: `<p>` elements can have only phrasing content.
Browsers treat current code as follows:
```html
<p></p>
<fieldset>...</fieldset>
<p></p>
```

Comment (teoli2003) [reply html/forms/form-validation/full-example.html:39]: Is there a change in mdn/content that has to be done?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
One question.

File: html/forms/form-validation/full-example.html (hunk 2 above)
Comment (OnkarRuikar) [reply html/forms/form-validation/full-example.html:39]: > Is there a change in mdn/content that has to be done?

Yes, https://github.com/mdn/content/pull/30901

File: html/forms/form-validation/full-example.html [hunk 3]
```diff
@@ -36,16 +36,14 @@
...
+      <legend>Do you have a driver's license?<span aria-label="required">*</span></legend>
+      <!-- While only one radio button in a same-named group can be selected at a time,
+           and therefore only one radio button in a same-named group having the "required"
+           attribute suffices in making a selection a requirement -->
+      <input type="radio" required name="driver" id="r1" value="yes"><label for="r1">Yes</label>
+      <input type="radio" required name="driver" id="r2" value="no"><label for="r2">No</label>
+    </fieldset>
```
Comment (teoli2003) [inline html/forms/form-validation/full-example.html:46]: Why the alignment here is different than the one on mdn/content?

Comment (OnkarRuikar) [reply html/forms/form-validation/full-example.html]: These bugs were preventing Prettier from formatting. So I pulled the fixes first thinking I would run Prettier later.

To make the fix whole let us commit the formatted file here itself:https://github.com/mdn/learning-area/pull/695/commits/6af44c9184c6939c4428bb8a226a171f34feb1b0

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by OnkarRuikar** [comment]
I don't think running Prettier on this repo is a good idea because we'll have to sync all the updated files with `mdn/content` repo which will be very painful.

**Comment by teoli2003** [comment]
I agree.

**Pull request merged**